tests/convergence/rates.cpp

```diff
@@ -26,6 +26,7 @@
 //               rates -m ../../data/inline-hex.mesh -sr 1 -prob 1 -o 2
 //               rates -m ../../data/square-disc.mesh -sr 2 -prob 1 -o 1
 //               rates -m ../../data/star.mesh -sr 2 -prob 3 -o 2
+//               rates -m ../../data/star.mesh -sr 2 -prob 3 -o 2 -j 0
```

Comment:
Wasn't `-j 0` the wrong option for diffusion? Or are you trying to preserve the autotests? I think this should demonstrate the correct option.

Reply:
`-j 0` is not the "natural" norm, so the rates for the jump term will be different from those in the preceding sample run. It's not necessarily "wrong" though.